helm_deploy/elite2-api/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
Is this second `.helmignore` required? Does the version that is down one directory not cover subdirectories?